api/scpca_portal/test/test_metadata_file.py

```diff
@@ -68,6 +96,10 @@ def test_write_metadata_dicts_read_write_values_match(self):
                 "country": "USA",
                 "language": "English",
                 "capital": "Washington DC",
+                "holidays": "New Year's Day;Birthday of Martin Luther King, Jr.;Inauguration Day;"
+                + "Washington's Birthday;Memorial Day;Juneteenth National Independence Day;"
+                + "Independence Day;Labor Day;Columbus Day;Veterans Day;Thanksgiving Day;"
+                + "Christmas Day",
```

Comment:
Try breaking up long strings this way instead
```suggestion
                "holidays": (
                    "New Year's Day;Birthday of Martin Luther King, Jr.;Inauguration Day;"
                    "Washington's Birthday;Memorial Day;Juneteenth National Independence Day;"
                    "Independence Day;Labor Day;Columbus Day;Veterans Day;Thanksgiving Day;"
                    "Christmas Day"
                  )
```

Reply:
See commit at fa899d8092b18f0d64284f5fef20819e15fe0e75.